[2022-02-12 14:01:18,547] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2022-02-12T17:01:08.870598+00:00 [queued]>
[2022-02-12 14:01:18,602] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2022-02-12T17:01:08.870598+00:00 [queued]>
[2022-02-12 14:01:18,603] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 14:01:18,603] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-02-12 14:01:18,607] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 14:01:18,646] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_2> on 2022-02-12T17:01:08.870598+00:00
[2022-02-12 14:01:18,662] {standard_task_runner.py:52} INFO - Started process 4667 to run task
[2022-02-12 14:01:18,677] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2022-02-12T17:01:08.870598+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_process.py', '--cfg-path', '/tmp/tmpbwnsbiim', '--error-file', '/tmp/tmpq4ap0pjs']
[2022-02-12 14:01:18,681] {standard_task_runner.py:77} INFO - Job 7: Subtask task_2
[2022-02-12 14:01:18,799] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2022-02-12T17:01:08.870598+00:00 [running]> on host ALE-LAPTOP.localdomain
[2022-02-12 14:01:19,077] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-02-12T17:01:08.870598+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-12T17:01:08.870598+00:00
[2022-02-12 14:01:19,080] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-02-12 14:01:19,081] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-02-12 14:01:19,096] {subprocess.py:75} INFO - Output:
[2022-02-12 14:01:22,102] {subprocess.py:83} INFO - Command exited with return code 0
[2022-02-12 14:01:22,159] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20220212T170108, start_date=20220212T170118, end_date=20220212T170122
[2022-02-12 14:01:22,226] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-12 14:01:22,270] {local_task_job.py:151} INFO - Task exited with return code 0
